docs/en/siem/machine-learning.asciidoc

```diff
@@ -20,12 +20,12 @@ image::ml-ui.png[]
 [float]
 [[manage-jobs]]
 === Manage machine learning jobs
-For users with the `ml_admin` role, the `Anomaly Detection` interface within
-the main navigation header can be used for for viewing, starting, and stopping
-SIEM machine learning jobs.
+For users with the `machine_learning_admin` role, the `ML job settings`
```

Comment:
Thanks for the distinction here as we've (myself at least 😅) been conflating the users/roles -- `ml_admin` is the user, with the roles being `machine_learning_admin`, `machine_learning_user`, etc.